fastlane_core/lib/fastlane_core/ui/github_issue_inspector_reporter.rb

```diff
@@ -0,0 +1,49 @@
+module Fastlane
+  # Adds all the necessary emojis (obv)
+  #
+  class InspectorReporter
+    NUMBER_OF_ISSUES_INLINE = 3
+
+    # Called just as the investigation has begun.
+    def inspector_started_query(query, inspector)
+      puts ""
+      puts "Looking for related GitHub issues on #{inspector.repo_owner}/#{inspector.repo_name}..."
+      puts "Search query: #{query}" if $verbose
+      puts ""
+    end
```

Comment:
I assume that because this is a delegate from above that these calls are all made asynchronously? Is there a case where some output will be printed to the terminal in between information from the inspector that could make it a little less clear what is going on than what's in your screenshots?